src/ui/qml/MainWindow.qml

```diff
@@ -39,16 +39,21 @@ ApplicationWindow {
             if (unreadCount > 0) {
                 if (audioNotifications !== null)
                     audioNotifications.message.play()
+                trayIcon.notifyMessage()
+                if (trayIcon.messagesEnabled) trayIcon.showText(qsTr("Event"), qsTr("New Message from")+" "+user.nickname)
```

Comment:
For future reference: you can't append/prepend text to translated strings, because it might not translate in the same order in all languages. Qt supports placeholders for this, like:

`qsTr("New message from %1").arg(user.nickname)`